js/index.ts

```diff
@@ -0,0 +1,306 @@
+
+export enum ROLES_ENUM    {
+    "RoleNobody" = "role:nobody",
```

Comment:
```suggestion
export enum Role {
    "Nobody" = "role:nobody",
```

В ROLES_ENUM начинать ключ с Role – это масло маслянное. Я бы сделал так как выше в примере. Как думаешь?

И в JS обычно Enum'ы именуют CamelCase'ом, в едином числе и без суфикса ENUM: https://www.typescriptlang.org/docs/handbook/enums.html, потому предлагаю переименовать.